src/rs/ffi/mod.rs

```diff
@@ -8,6 +8,50 @@
 
 pub type QUIC_ADDR = std::ffi::c_void;
 
+#[cfg(target_os = "windows")]
+pub type HANDLE = std::os::windows::raw::HANDLE;
+
+#[cfg(target_os = "windows")]
+#[repr(C)]
+pub struct OVERLAPPED {
```

Comment:
U can allowlist the bindgen types by add them here:
https://github.com/microsoft/msquic/blob/58fd56c6ba8b8190042660ca83d7748b7a1ca558/scripts/build.rs#L210C1-L210C61

This saves the manual work here but can get a lot more generated code.
I originally blocklisted QUIC_ADDR  because it creates too much code for manipulating socket_addr. 
Maybe we should allowlist all types and get rid of the c_types dependencies.